src/status_im/ui2/screens/common/contact_list/view.cljs

```diff
@@ -0,0 +1,27 @@
+(ns status-im.ui2.screens.common.contact-list.view
+  (:require [react-native.core :as rn]
+            [quo2.foundations.colors :as colors]
+            [status-im2.common.contact-list-item.view :as contact-list-item]
+            [quo2.core :as quo2]
+            [utils.re-frame :as rf]))
+
+(defn contacts-section-header [{:keys [title]}]
+  [rn/view {:style {:border-top-width   1
+                    :border-top-color   colors/neutral-20
+                    :padding-vertical   8
+                    :padding-horizontal 20 :margin-top 8}}
+   [quo2/text {:size   :paragraph-2
+               :weight :medium
+               :style  {:color colors/neutral-50}} title]])
+
+(defn contact-list [data]
+  (let [contacts (rf/sub [:contacts/filtered-active-sections])]
+    [rn/section-list
+     {:key-fn                         :title
+      :sticky-section-headers-enabled false
+      :sections                       contacts
+      :render-section-header-fn       contacts-section-header
+      :content-container-style        {:padding-bottom 120}
+      :sticky-header-indices [0]
+      :render-fn                      (fn [item]
+                                        [contact-list-item/contact-list-item item data])}]))
```

Comment:
better to use `render-data` property

```clojure
:render-data data
:render-fn  contact-list-item/contact-list-item 
```

Reply:
@flexsurfer, could you expand on why :render-data is better? It's not clear to me. Is it because of what wrap-render-fn does behind the scenes?

Reply:
@flexsurfer what is `render-data` property?

Reply:
render-fn shouldn't anonymous because it will be recreated each time on every contacts change, and for list it will be a new function each time, so render-fn should be a defined function like `:render-fn   contact-list-item/contact-list-item` but sometimes you want to pass additional data to render function that's why we added additional property `:render-data` so you can pass data safely to render function, it will be a fourth argument in the render function